### PR TITLE
fix: rmrk search issue

### LINKED ISSUE #8759: rmrk search issue
Capture d’écran 2023-12-26 à 7 59 41 AM

- chain `rmrk`
- search `green surfing`
- no result in searchbar
- result in marketplace

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8759

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

https://github.com/kodadot/nft-gallery/assets/44554284/c4c19484-38be-4faa-b778-ae2c776ef602

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

File: composables/useSearchNfts.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+export const queryPrefix = {
+  ksm: 'chain-ksm',
+  rmrk: 'chain-rmrk',
+}
```
Comment (roiLeo) [inline composables/useSearchNfts.ts:4]: move to utils?

**Review by roiLeo — APPROVED** [review]
can't we use `useSearchNfts` composable instead?

**Comment by hassnian** [comment]
> can't we use `useSearchNfts` composable instead?

done

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 8.65 USD/DOT ~ 3.468 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x588008d717e814cc98d84dabde9df515816b8d3b90cb981fcae32dd1474e5394](https://polkadot.subscan.io/extrinsic/0x588008d717e814cc98d84dabde9df515816b8d3b90cb981fcae32dd1474e5394)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**